typhon/cloudmask/aster.py

```diff
@@ -617,79 +619,100 @@ def sensor_angles(self, sensor="vnir"):
              Algorithm Theoretical Basis Document for ASTER Level-1 Data 
              Processing (Ver. 3.0).1996.
         """
+        if channel is not "3B":
```

Comment:
[You use `==` when comparing values and `is` when comparing identities.](https://stackoverflow.com/a/2988117/10544367)

Reply:
Good point.Thanks.